android/src/main/cpp/graphics/shader/ColorShaderOpenGl.cpp

```diff
@@ -29,8 +29,10 @@ void ColorShaderOpenGl::setupProgram(const std::shared_ptr<::RenderingContextInt
     glDeleteShader(vertexShader);
     glAttachShader(program, fragmentShader); // add the fragment shader to program
     glDeleteShader(fragmentShader);
+    glCheckError();
```

Comment:
Für die Konsistenz mit den anderen Shadern müsste man diese entweder um die Checks ergänzen oder aber diese vorerst entfernen. Aktuell ist der Kompromiss zu einem vollständigeren check in allen OpenGL Klassen, dass es im Renderer pro Frame einmal geprüft wird, um wenigstens grobmaschig keinen Error zu verpassen.

Reply:
Done. Ich habe dafür den hier fehlenden call zu `checkGlProgramLinking` dazu genommen.